### PR TITLE
Add Vecosek 0.0.0 (3 packages)

### PR DESCRIPTION
**Pull request by smondet**

### CODE REVIEW

File: packages/vecosek-engine/vecosek-engine.0.0.0/opam [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+]
+homepage: "https://gitlab.com/smondet/vecosek"
+bug-reports: "https://gitlab.com/smondet/vecosek/issues"
+dev-repo: "https://gitlab.com/smondet/vecosek.git"
+license: "ISC"
+version: "0.0.0"
+available: [ ocaml-version >= "4.03.0" ]
+build: [
+  ["ocaml" "please.ml" "configure"]
+  ["jbuilder" "build" "--only" "vecosek-engine" "--root" "." "-j" jobs "@install"]
```
Comment (kit-ty-kate) [inline packages/vecosek-engine/vecosek-engine.0.0.0/opam]: This line look quite different from the usual one. Is there any reason for that ? (see almost any other packages using jbuilder for examples)

File: packages/vecosek-engine/vecosek-engine.0.0.0/opam [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+]
+depends: [
+  "vecosek-scene"
```
Comment (kit-ty-kate) [inline packages/vecosek-engine/vecosek-engine.0.0.0/opam]: jbuilder is not listed as dependency

**Comment by smondet** [comment]
Travis failed on OSX this time:

```
=-=- Gathering sources =-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=  🐫 
[WARNING] download of
          http://download.camlcity.org/download/findlib-1.8.0.tar.gz failed,
          trying mirror
[ERROR] The sources of the following couldn't be obtained, aborting:
          - atd.1.12.0
          - atdgen.1.12.0
          - biniou.1.2.0
          - cppo.1.6.4
          - easy-format.1.3.1
          - jbuilder.1.0+beta20
          - menhir.20171222
          - nonstd.0.0.3
          - ocamlbuild.0.12.0
          - ocamlfind.1.8.0
          - sosa.0.3.0
          - vecosek-scene.0.0.0
          - yojson.1.4.1
        (This may be fixed by running 'opam update')
```

(but it succeeded earlier in this PR, which is amazing given that I've never tried this project on Mac OSX!)

**Comment by samoht** [comment]
Thanks!

**Pull request merged**